_data/vendors.yml

```diff
@@ -333,6 +333,15 @@
   updated_at: 2018-10-17
   footnotes: "[^victorops]: You can add SSO to a plan for $5 per u/m"
 
+- name: Whitesource
+  url: https://www.whitesourcesoftware.com/
+  base_pricing: $250 per month
+  sso_pricing: $3,0000 per year
```

Comment:
Might want to change this to $28,000, to reflect pricing changes and increase legibility.